src/static/toplevel.jl

```diff
@@ -19,7 +19,7 @@ struct ToplevelCall <: ToplevelItem
     callstr::String
 end
 
-struct ToplevelTupleH <: ToplevelItem
+struct ToplevelModuleUsage <: ToplevelItem
```

Comment:
please let me know if you have suggestions for this naming

Reply:
This is fine, imho.